### PR TITLE
Use CSS zoom to scale card content

### PR DESCRIPTION
**Pull request by hssm**
I am replacing the `Reviewer font scaling` setting with a `Card zoom` setting instead.

When users make use of the `Reviewer font scaling` setting, I am almost certain they mean to say
> Zoom my cards in/out by this much please

...and definitely don't mean
> Resize my divs and spans by this much but *only if* my stylesheet has a `font-size` rule for them

The entire `Reviewer font scaling` feature can be replaced with a line of CSS, so I've done that. This removes a rather bulky processing step from our card display. In terms of card display speed, it will likely only be perceptible to those with both very large templates and slow(ish) devices since it still isn't really where our bottleneck is.

Other things:
1 - The preferences will be upgraded for users on the next version bump (I haven't done the version bump)
2 - I've also renamed `image scaling` to `image zoom`.
3 - I've moved both of these features out of the `Fonts` category and into the `Reviewing -> Display` category. There still remains the `card browser font scaling` setting inside `Fonts` which I feel is out of place. Not sure where else to put it.

### CODE REVIEW

**Comment by timrae** [comment]
What would happen in the case where the card already uses CSS zoom? For instance the user sets the zoom in AnkiDroid settings, and then downloads a shared deck that already makes use of it for some reason.

**Comment by hssm** [comment]
It'll still zoom as you would expect. You can have [nested zoom levels](http://jsfiddle.net/n19v3vnr/) and elements will scale relative to their parent's size.

The only case where this "breaks" is if the user's stylesheet also targets the `body` element on the `zoom` property where it will overwrite it. I expect user stylesheets to target `card` in the first place so I don't think of this as a big drawback.

File: AnkiDroid/src/main/res/values/10-preferences.xml [hunk 1]
```diff
@@ -56,14 +56,14 @@
     <string name="reset_languages_summ">Reset language assignments (for text to speech and dictionaries) for all decks. Reset specific deck assignments on deck picker.</string>
     <string name="reset_languages_question">Reset all language assignments?</string>
     <string name="reset_confirmation">All states reset</string>
-    <string name="display_font_size">Reviewer font scaling</string>
-    <string name="display_font_size_summ">XXX %</string>
-    <string name="image_size">Reviewer image scaling</string>
-    <string name="image_size_summ">XXX %</string>
+    <string name="card_zoom">Card zoom</string>
+    <string name="card_zoom_summ">XXX%</string>
```
Comment (timrae) [inline AnkiDroid/src/main/res/values/10-preferences.xml]: I think these `XXX%` strings don't require translations right? If so, they can be moved to the constants file

Comment (ospalh) [inline AnkiDroid/src/main/res/values/10-preferences.xml]: I don`t know.
How about Arabic/Hebrew/? Maybe keep them.
Afaiks the single "< " that is added to the review times for "< 10 minuets" *should* be translated for Arabic

File: AnkiDroid/src/main/res/values/10-preferences.xml [hunk 2]
```diff
@@ -56,14 +56,14 @@
...
+    <string name="image_zoom">Image zoom</string>
+    <string name="image_zoom_summ">XXX%</string>
     <string name="button_size">Answer button size</string>
-    <string name="button_size_summ">XXX %</string>
+    <string name="button_size_summ">XXX%</string>
```
Comment (timrae) [inline AnkiDroid/src/main/res/values/10-preferences.xml]: I think these 4 `XXX%` strings don't require translations right? I moved them out to a constant in #877

Comment (timrae) [inline AnkiDroid/src/main/res/values/10-preferences.xml]: @ospalh sorry I missed your reply before I deleted my original comment, let's discuss at #877

**Pull request merged**